packages/python-pulpcore/python-pulpcore.spec

```diff
@@ -147,38 +103,58 @@ Using Pulp you can:
 - Manage content from multiple sources in one place
 - Promote content through different repos in an organized way
 
+
 %prep
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
 %autosetup -n %{pypi_name}-%{version}
 # Remove bundled egg-info
 rm -rf %{pypi_name}.egg-info
 
+# relax cryptography requirement
+# see https://pulp.plan.io/issues/9367
+sed -i '/cryptography/ s/~=.*//' requirements.txt
+%{?scl:EOF}
+
+
 %build
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
 %py3_build
+%{?scl:EOF}
 
 for wrapper in %{wrappers}
 do
-  printf '#!/bin/bash\nexec %s "$@"\n' ${wrapper} > ${wrapper}
+  printf '#!/bin/bash\n%{?scl:source scl_source enable tfm-pulpcore \n}exec %s "$@"\n' ${wrapper} > ${wrapper}
 done
 
+
 %install
+%{?scl:scl enable %{scl} - << \EOF}
+set -ex
 %py3_install
+%{?scl:EOF}
 
 for wrapper in %{wrappers}
 do
-  install -D -m 755 ${wrapper} %{buildroot}%{_libexecdir}/%{pypi_name}/${wrapper}
+  install -D -m 755 ${wrapper} %{buildroot}%{_root_libexecdir}/%{pypi_name}/${wrapper}
 done
 
-%files -n python%{python3_pkgversion}-%{pypi_name}
+%files -n %{?scl_prefix}python%{python3_pkgversion}-%{pypi_name}
 %license LICENSE
 %doc README.md
 %{_bindir}/pulp-content
 %{_bindir}/pulpcore-manager
 %{_bindir}/pulpcore-worker
```

Comment:
This puts these into the SCL'ed bindir.

For the worker, we use the libexec wrapper anyways, so it shouldn't hurt.
`pulpcore-manager` is called by `puppet-pulpcore` -- should we place a wrapper in `/usr/bin` that does the SCL dance for us?
And no idea where/how `pulp-content` is used.

CC @ekohl

Reply:
Pulp's Ansible-based installer also looks for a `/usr/bin/pulpcore-manager`, so I guess a wrapper is in order.

Reply:
:+1: for a wrapper that activates the SCL if needed. That makes it easy for the installer(s) and users.

I'd also love it if `pulpcore-manager` on packaging actually set a default for `PULP_SETTINGS` but that's probably out of scope.

Reply:
added.